rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf

```diff
@@ -1343,6 +1343,38 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|!REQUEST_COOKIES:/_pk_ref/|REQU
     setvar:'tx.%{rule.id}-OWASP_CRS/WEB_ATTACK/SQL_INJECTION-%{MATCHED_VAR_NAME}=%{tx.0}'"
 
 
+#
+# -=[ LibInjection Check on last path segment ]=-
+#
+# This is a sibling of rule 942100 that adds checking of the last path segment.
+#
+SecRule REQUEST_FILENAME "@rx ^/+(.*)$" \
+    "id:942101,\
+    phase:2,\
+    block,\
+    capture,\
+    t:none,t:utf8toUnicode,t:urlDecodeUni,t:removeNulls,\
+    msg:'SQL Injection Attack Detected via libinjection',\
+    logdata:'Matched Data: %{TX.0} found within %{MATCHED_VAR_NAME}: %{MATCHED_VAR}',\
+    tag:'application-multi',\
+    tag:'language-multi',\
+    tag:'platform-multi',\
+    tag:'attack-sqli',\
+    tag:'OWASP_CRS/WEB_ATTACK/SQL_INJECTION',\
+    tag:'WASCTC/WASC-19',\
+    tag:'OWASP_TOP_10/A1',\
+    tag:'OWASP_AppSensor/CIE1',\
+    tag:'PCI/6.5.2',\
+    tag:'paranoia-level/2',\
+    ver:'OWASP_CRS/3.1.0',\
+    severity:'CRITICAL',\
+    chain"
+    SecRule TX:1 "@detectSQLi" \
+        "setvar:'tx.anomaly_score_pl1=+%{tx.critical_anomaly_score}',\
```

Comment:
If this is PL2 you need to increment pl2.

Reply:
I don't understand what you mean. The chain starter rule has `tag:'paranoia-level/2'` and is located within the PL2 section...

Reply:
Fixed, thanks.